xbmcaddon.py

```diff
@@ -220,15 +244,46 @@ def __get_strings(self):
 
         return translations
 
+    def __filter_lang_xmltag(self, tag, xml_content):
+        tag_matches = re.findall(r'<' + tag + r'(?:\s*lang=\"([a-zA-Z-]+)\")?\s*>(.*?)</' + tag + '>', xml_content, flags=re.DOTALL)
+        if tag_matches:
+            for match in tag_matches:
+                if match[0].startswith('en'):
+                    return match[1]
+            else:
+                return tag_matches[0][1]
+        return None
+
     def __load_add_on_xml(self):
         add_on_xml = os.path.join(self.__add_on_path, "addon.xml")
 
         with io.open(add_on_xml, encoding='utf-8') as fp:
             xml_content = fp.read()
-            self.__version = re.findall(r'version="(\d+.\d+.\d+[^"]*)', xml_content)[0]
+            self.__version = re.findall(r'<addon.*?version="(\d+(?:\.\d)?(?:\.\d)?[^"]*)', xml_content, flags=re.DOTALL)[0]
             self.__add_on_id = re.findall(r'addon\W+id="([^"]+)"', xml_content)[0]
```

Comment:
This one fails if the `id` is not the first attribute. Could you perhaps fix this and make it similar to the others: `re.findall(r'addon.*?id="([^"]+)"', xml_content, flags=re.DOTALL)[0]`